mautic/mautic
# [UX] Improve saving options

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Modern apps tend to use only "save/cancel" to simplify the user experience and keep the interface clean and intuitive. This PR ensures that the system performs the appropriate action in the user context.

When clicking Save, the system will automatically close the editing screen in these scenarios:

- Contacts
- Companies
- Forms
- Email
- Dynamic Content
- Campaigns
- Categories
- Tags
- Reports
- Stages
- Points (Actions, Triggers and Groups)
- Segments
- Landing pages
- Assets
- Roles
- Users
- Focus Items
- Mobile Notifications
- Web Notifications
- API
- SMS
- Custom Fields
- Plugins

In these other scenarios, the system will simply apply the changes without taking the user out of the page:

- Configuration
- Account

Why must we change that?

1. UX research indicates that many options can overwhelm users, leading to indecision or even inaction, a phenomenon known as analysis paralysis. 
2. An excessive number of choices increases cognitive load, making the decision-making process more complex and tiresome for the user. 
3. Reducing options can enhance efficiency, allowing users to complete their tasks more quickly and with less effort. 
4. Studies show that, while users may initially appreciate having many options, they often feel more satisfied when the decision-making process is straightforward and direct. 
5. Simpler interfaces are easier to learn and remember, which is especially important for new users or those who use the system sporadically. 
6. Maintaining consistency with recognized design patterns helps users anticipate system behavior, which can be disrupted by the introduction of additional options.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Tags, for example, > New and see you have only Save and Cancel
